sympy/simplify/simplify.py

```diff
@@ -1270,57 +1281,74 @@ def nsimplify_real(x):
             im = nsimplify_real(im)
     except ValueError:
         if rational is None:
-            return _real_to_rational(expr)
+            return _real_to_rational(expr, rational_alg=rational_alg)
         return expr
 
     rv = re + im*S.ImaginaryUnit
     # if there was a change or rational is explicitly not wanted
     # return the value, else return the Rational representation
     if rv != expr or rational is False:
         return rv
-    return _real_to_rational(expr)
+    return _real_to_rational(expr, rational_alg=rational_alg)
 
 
-def _real_to_rational(expr, tolerance=None):
+def _real_to_rational(expr, tolerance=None, rational_alg='base10'):
```

Comment:
Maybe yet another name could be found for the keyword. To me, it suggests both algebra and algorithm. Perhaps `conversion` could be used?